frontend/src/components/Card.tsx

```diff
@@ -99,7 +99,7 @@ const Card = ({
               ))}
           </div>
           {projectName && (
-            <a href={projectLink} target="_blank" rel="noopener noreferrer">
+            <a href={projectLink} target="_blank" rel="noopener noreferrer" className="font-bold">
```

Comment:
@harsh3dev Longer titles don't work with this layout right now: 
![Screenshot 2024-12-22 at 9 30 56 AM](https://github.com/user-attachments/assets/335b40a6-69f9-4b4a-9b84-1f1aea4c674b)
Could you update it and make sure that other pages still look good?

Reply:
I will update this after updating the existing issues.
Thanks for reviewing.